### ISSUE TITLE
Target group issue when using pre-existing public-facing ALB

### ISSUE THREAD

**Issue by kreiza** [aws/copilot-cli#5485]
Hi! I'm getting such an error when I'm trying to use my existing ALB (it has 1 listener on port 80)
ValidationError: Template format error: Unresolved resource dependencies [TargetGroup] in the Resources block of the template
What could be the problem?

**Comment by huanjani** [comment]
Hi @kreiza! 

Sorry about this bug; we also discovered it yesterday. We have a PR open to fix it: https://github.com/aws/copilot-cli/pull/5478

We will get that merged in as soon as possible. You can 1. [longer wait] wait for the fix to be released, 2. [shorter wait] build the binary from the mainline branch of the Copilot repository after the PR is merged, or 3. [no wait] use a YAML patch to fix it with a YAML patch:

1. run `copilot svc override` and choose the yaml patch option
2. in the generated `cfn.patches.yml` file, add:
```
- op: replace
  path: /Resources/HTTPListenerRuleForImportedALB/Properties/Actions/0/TargetGroupArn
  value: !Ref TargetGroupForImportedALB
```
(This is assuming you don't have multiple rules; if you do, you'll need to add indices and do a "replace" for each one.)
3. run `copilot svc deploy` again

Please let us know if that works for you! Thanks for the issue and apologies for the bug!

**Comment by kreiza** [comment]
@huanjani Thanks a lot for your help and effort, really appreciate it

**Comment by iamhopaul123** [comment]
Hello! The fix is now released in v1.32.1: https://github.com/aws/copilot-cli/releases/tag/v1.32.1
